projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { Text } from '@automattic/jetpack-components';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import ProductCard from '../../connected-product-card';
+import useVideoPressCardDescription from './use-videopress-description';
+import VideoPressValueSection from './videopress-value-section';
+import type { ProductCardType } from '../types';
+
+import './style.scss';
+
+const VideopressCard: ProductCardType = ( { admin } ) => {
+	const slug = 'videopress';
```

Comment:
We could consider putting static variables outside of component scope?

Reply:
I don't think it matters too much but we definitely could do that 🤔 And it'd be easy to do it now before we get too far into the product card projects